### PR TITLE
Add IPFabric Integration

### PR DESCRIPTION
**Pull request by snaselj**
# Closes NaN

## What's Changed

Added IPFabric integration from develop branch.

### CODE REVIEW

**Comment by snaselj** [comment]
This PR is ready for review now. All IPFabric unittests are passing, however it needs to be tested manually.

@jdrew82

**Comment by pke11y** [comment]
@alhogan can you help with the testing here?

**Comment by alhogan** [comment]
> @alhogan can you help with the testing here?

Absolutely, I will get this done today.

File: nautobot_ssot/__init__.py [hunk 1]
```diff
@@ -70,6 +71,11 @@ class NautobotSSOTPluginConfig(PluginConfig):
         "infoblox_username": "",
         "infoblox_verify_ssl": True,
         "infoblox_wapi_version": "",
+        "ipfabric_api_token": "",
+        "ipfabric_host": "",
+        "ipfabric_ssl_verify": True,
+        "ipfabric_timeout": 15,
+        "nautobot_host": "",
```
Comment (jdrew82) [inline nautobot_ssot/__init__.py:78]:
```suggestion
        "ipfabric_nautobot_host": "",
```

**Review by jdrew82 — APPROVED** [review]
LGTM!

**Pull request merged**